yarn.lock

```diff
@@ -17599,4 +17599,8 @@ zksync-web3@^0.4.0:
 zod@^3.11.6:
   version "3.17.3"
   resolved "https://registry.yarnpkg.com/zod/-/zod-3.17.3.tgz#86abbc670ff0063a4588d85a4dcc917d6e4af2ba"
+<<<<<<< HEAD
```

Comment:
Please revert these changes.

Reply:
Sorry about that. It was pulled from master though, do you think my yarn changed that in the few minutes I had it in my repo?

Reply:
You had a conflict during rebasing and you didn't resolve it. You can just drop the changes in this file.